pkg/cluster/backupstorage/s3.go

```diff
@@ -15,6 +15,8 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
+const awsConfigDirPrefix = "/var/tmp/etcd-operator/aws"
```

Comment:
probably we should define /var/tmp/etcd-operator as "operatorRoot" somewhere in our codebase.